common/src/main/java/com/revenuecat/purchases/common/purchaseExtensions.kt

```diff
@@ -5,21 +5,19 @@ import com.revenuecat.purchases.strings.BillingStrings
 import java.util.ArrayList
 
 fun Purchase.toHumanReadableDescription() =
-    "skus: ${
-        this.skus.joinToString(prefix = "[", postfix = "]")
+    "productIds: ${
+        this.products.joinToString(prefix = "[", postfix = "]")
     }, orderId: ${this.orderId}, purchaseToken: ${this.purchaseToken}"
 
 /**
  * **Important**: until multi-line subscriptions are released,
  * we assume that there will be only a single sku in the purchase.
  * https://android-developers.googleblog.com/2021/05/whats-new-in-google-play-2021.html
  */
-val Purchase.firstSku: String
-    get() = skus[0].also {
-        if (skus.size > 1) {
+val Purchase.firstProductId: String
+    get() = products[0].also {
+        if (products.size > 1) {
             log(LogIntent.GOOGLE_WARNING, BillingStrings.BILLING_PURCHASE_MORE_THAN_ONE_SKU)
         }
     }
 
-val Purchase.listOfSkus: ArrayList<String>
```

Comment:
i didn't really get what this did for us...

Reply:
I think this was required to have BC3 and BC4 living together in harmony. In BC3 it returned something different than in BC4, but the code was agnostic, so I created a `listOfSkus` extension. It can be removed now we don't have multiple flavor